sdk/personalizer/azure-ai-personalizer/samples/README.md

```diff
@@ -0,0 +1,47 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-ai-personalizer
```

Comment:
I think the metadata here should just be `azure-personalizer` looking at the allowed values: https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main